src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
As far as I understand pointers on C# is bad and Marshalling is the native way of handling with it. Given that and the fact that SendMessage changes the semantics of wParam and lParam according to the WIndows Message you are sending (some use it for storing numbers/values directly without using the pointer, others use as pointers to structures, others to pointers to receive characters buffers to be changed, and so on) I think overloads are the best option to make it easier to the user/developer that is not so acquainted with native-managed marshalling and all the little quirks that it comes with.

If you see the understanding of LPARAM and WPARAM from Microsoft developers you will notice that it is a naming reminiscent of old platform support (16-bit Windows). 

> When Windows was converted to 32-bit, the WPARAM parameter grew to a 32-bit value as well. So even though the “W” stands for “word”, it isn’t a word any more. (And in 64-bit Windows, both parameters are 64-bit values!)
> 
> It is helpful to understand the origin of the terms. If you look at the design of window messages, you will see that if the message takes a pointer, the pointer is usually passed in the LPARAM, whereas if the message takes a handle or an integer, then it is passed in the WPARAM. (And if a message takes both, the integer goes in the WPARAM and the pointer goes in the LPARAM.)

 [What do the letters W and L stand for in WPARAM and LPARAM?](https://blogs.msdn.microsoft.com/oldnewthing/20031125-00/?p=41713)

Given that, actually, the uint wParam, int lParam is the underlying implementation on Windows. The IntPtr marshalling helps us for pointers but it doesn't make sense to box/unbox it just for the sake of using IntPtr, at least for some of the Windows Messages that uses only values and not pointers.